tests/backends/test_rdflib.py

```diff
@@ -5,13 +5,15 @@
 
 import pytest
 
-from tripper import Literal, Triplestore
+from tripper import RDFS, Literal, Triplestore
 
 rdflib = pytest.importorskip("rdflib")
 
 
-# Test for issue #162: Literals are lost when listing triples with rdflib
 ts = Triplestore("rdflib")
+EX = ts.bind("", "http://ex#")
+
+# Test for issue #162: Literals are lost when listing triples with rdflib
```

Comment:
This comment does not make sense. Are we testing that triples listed with rdflib includes literals?

Reply:
Agree, the comment is badly phrased. Rephrased it to:

    # Test for issue #162: Literals are lost when accessing triples with rdflib